README.md

```diff
@@ -90,9 +84,13 @@ You can disable the slider to prevent the user from moving it.
   disabled: PropTypes.bool
 ```
 
-## Usage
+`react-with-styles` adds a styles object via a higher-order component.
```

Comment:
Should styles be in backticks here, since it references code?

Also, it might be worth making it very clear that this isn't exactly part of the public API. Maybe also include a link to the "Using react-with-styles" section here?